Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -18,16 +18,16 @@
 #include <GraphMol/MolOps.h>
 #include <GraphMol/MolStandardize/TransformCatalog/TransformCatalogParams.h>
 #include "Charge.h"
-
 #include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
-using namespace std;
 
+using namespace std;
 namespace RDKit {
 namespace MolStandardize {
 const CleanupParameters defaultCleanupParameters;
 
-RWMol *cleanup(const RWMol &mol, const CleanupParameters &params) {
+RWMol *cleanup(const RWMol &mol, const CleanupParameters &params) 
```

Comment:
This formatting is different from what we normally use. Please switch it back .

Reply:
The problem is with the bracket, right? I am guessing you want it to be on the same line.

Reply:
Right

Reply:
Right